src/ensembl/src/shared/components/feature-summary-strip/FeatureSummaryStrip.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+import { GeneSummaryStrip, RegionSummaryStrip } from '../feature-summary-strip';
+
+import { EnsObject } from 'src/shared/state/ens-object/ensObjectTypes';
+
+type FeatureSummaryStripProps = {
+  ensObject: EnsObject | null;
```

Comment:
This component hardly makes sense if ensObject is `null`. Perhaps better to declare that in `FeatureSummaryStripProps` `ensObject` is always non-nullable, and check for null in the parent component that renders the `FeatureSummaryStrip`?